### DISCUSSION TITLE
Invalid list for option 'additional_hosts' in Cloudflared (9074a9fa_cloudflared). Got {'external_hostname': '', 'additional_hosts': 'homeassistant.1921686871.work'}

### DISCUSSION THREAD

**Discussion by Jetpilot1985** [homeassistant-apps/app-cloudflared#345] [Q&A] [answered]
Hello!

When I enter the the host name I get the following Error: (German) 

Fehler beim Speichern der Add-on Konfiguration, Invalid list for option 'additional_hosts' in Cloudflared (9074a9fa_cloudflared). Got {'external_hostname': '1921686871.work', 'additional_hosts': {}}

Can someone help me please?

**Comment by elcajon** [comment, marked as answer]
Take a look [here](https://github.com/brenner-tobias/addon-cloudflared/wiki/Troubleshooting#remove-additional_hosts-option)

**Comment by Teacup42** [comment]
I had this same error. Your additional host, did you use HTTP or HTTPS ? For me HTTPS did the trick.

See for more info: https://community.home-assistant.io/t/cloudflared-unable-to-save-additional-hosts/553059
